corehq/apps/data_interfaces/interfaces.py

```diff
@@ -91,6 +105,41 @@ def rows(self):
                 naturaltime(parse_date(es_case['modified_on'])),
             ]
 
+    @property
+    def bulk_response(self):
+        from .views import BulkCaseReassignSatusView
+        from .tasks import bulk_case_reassign_async
+        owner_id = self.request_params.get('new_owner_id', None)
+        if not owner_id:
+            return HttpResponseBadRequest(
+                _("An owner_id needs to be specified to bulk reassign cases")
+            )
+
+        case_ids = [
+            self.get_case(row)['_id']
+            for row in self.es_results['hits'].get('hits', [])
+        ]
+
+        task_ref = expose_cached_download(
+            payload=case_ids, expiry=60 * 60, file_extension=None
+        )
+        task = bulk_case_reassign_async.delay(
+            self.domain,
+            self.request.couch_user.get_id,
+            owner_id,
+            task_ref.download_id,
+            self.request.META['HTTP_REFERER']
+        )
+        task_ref.set_task(task)
+        if self.request.method != 'POST':
```

Comment:
Maybe a nit, but it feels awkward to only be concerned about the request method at this stage of the method, since the user will see an error but the reassignment process is kicked off in the background. Should this be at the top of this method or is it placed here intentionally?

Reply:
Ah, this is supposed to be above. I will update it